src/api/apiFetch.js

```diff
@@ -27,7 +28,7 @@ export const apiCall = async (
   auth: Auth,
   route: string,
   params: Object = {},
-  resFunc = res => res,
+  resFunc: (Json) => mixed = res => res,
```

Comment:
Nope it isn't reall :)
Also not sure this is even correct way to declare this. My guess is not.